src/parser/SparqlParser.cpp

```diff
@@ -322,18 +322,10 @@ void SparqlParser::parseWhere(ParsedQuery* query,
         // Add the variables from the subquery that are visible to the outside
```

Comment:
The line above (where you get the `_varsOrAsterisk`) should be below the comment.